packages/core/lib/command-utils.js

```diff
@@ -201,6 +203,37 @@ const runCommand = async function (command, options) {
   return await command.run(options);
 };
 
+const processHelpInput = inputStrings => {
+  //When we think user is trying to access help incorrectly, provide them with some helpful information.
+  if (inputStrings.length > 2 && ["help", "--help"].includes(inputStrings[1])) {
+    const help = require("./commands/help").meta;
+    console.log(
+      "Error: please use the following syntax for accessing help information\n",
+      "\n               ",
+      help.help.usage
```

Comment:
instead of just displaying the usage for the help command, shouldn't we also tell them about the other formats we want to accept?

Reply:
It does provide that syntax info in the usage. 
```truffle migrate --help helo
Error: please use the following syntax for accessing help information

                truffle help [<command> <subCommand>]
                truffle --help [<command> <subCommand>]
                truffle [<command> <subCommand>] help
                truffle [<command> <subCommand>] --help